frontend/src/types.d.ts

```diff
@@ -58,6 +60,8 @@ declare global {
     process?: {
       type?: string
     }
+    analytics: SegmentAnalytics.AnalyticsJS
+    ga: S
```

Comment:
What is `S`?

Reply:
left over from some initial trials.  Removed.